assignments/closure.js

```diff
@@ -1,21 +1,64 @@
 // ==== Challenge 1: Write your own closure ====
 // Write a simple closure of your own creation.  Keep it simple!
 
+function sayHello() {
+  let greet = "Hello";
 
-/* STRETCH PROBLEMS, Do not attempt until you have completed all previous tasks for today's project files */
+  function sayWorld() {
+    console.log(greet, "World!");
```

Comment:
```suggestion
    return `${greet}, World!`
```
Nice closure, but I recommend doing this instead here